samples/apps/copilot-chat-app/webapp/src/components/chat/invitation-dialog/InvitationJoinDialog.tsx

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import { useMsal } from "@azure/msal-react";
+import { Button, DialogActions, DialogBody, DialogContent, DialogSurface, DialogTitle, Divider, Input, Label, makeStyles, tokens } from "@fluentui/react-components";
+import React from "react";
+import { AuthHelper } from "../../../libs/auth/AuthHelper";
+import { ChatService } from "../../../libs/services/ChatService";
+import { useChat } from "../../../libs/useChat";
+
+const useStyles = makeStyles({
+    content: {
+        display: "flex",
+        flexDirection: "column",
+        rowGap: tokens.spacingVerticalMNudge,
+    },
+    divider: {
+        marginTop: tokens.spacingVerticalXL,
+        marginBottom: tokens.spacingVerticalXL,
+    },
+});
+
+interface InvitationJoinDialogProps {
+    onCloseDialog: () => void;
+}
+
+export const InvitationJoinDialog: React.FC<InvitationJoinDialogProps> = ({ onCloseDialog }) => {
+    const { instance, inProgress } = useMsal();
+    const account = instance.getActiveAccount();
+    const chatService = new ChatService(process.env.REACT_APP_BACKEND_URI as string);
+    const chat = useChat();
+    const [errorOccurred, setErrorOccurred] = React.useState<boolean>(false);
+    const [errorMessage, setErrorMessage] = React.useState<string>("");
+
+    const classes = useStyles();
+
+    const handleSubmit = async (ev: React.FormEvent<HTMLFormElement>) => { 
+        ev.preventDefault();
+        setErrorOccurred(false);
+
+        const chatId = ev.currentTarget.elements.namedItem("chat-id-input") as HTMLInputElement;
```

Comment:
Would change this to track a string state value, querying DOM elements can get expensive

Reply:
I believe this is only traversing the form, which has limited controls.